manifests/init.pp

```diff
@@ -0,0 +1,10 @@
+class robottelo_slave () {
```

Comment:
I realize none of the classes are very complicated but I do like class headers (I mean these: https://github.com/Katello/puppet-qpid/blob/master/manifests/init.pp#L1-L48)  regardless, it reminds future authors to update them especially if things are parameterized later